airflow/cli/commands/dag_command.py

```diff
@@ -145,6 +145,11 @@ def dag_backfill(args, dag: list[DAG] | DAG | None = None) -> None:
 @cli_utils.action_cli
 def dag_trigger(args) -> None:
     """Creates a dag run for the specified dag."""
+    dag = DagModel.get_dagmodel(args.dag_id)
+    if not dag:
+        raise SystemExit(f"DAG: {args.dag_id} does not exist in 'dag' table")
+    if dag.is_paused:
+        raise AirflowException(f"DAG with dag_id: {args.dag_id} is paused.")
```

Comment:
Why do these raise different exception classes?

Reply:
it shouldn't. thank you for pointing out.